src/components/ids-multiselect/ids-multiselect.ts

```diff
@@ -0,0 +1,270 @@
+import { customElement, scss } from '../../core/ids-decorators';
+import { attributes } from '../../core/ids-attributes';
+
+import '../ids-checkbox/ids-checkbox';
+import '../ids-dropdown/ids-dropdown';
+import Base from './ids-multiselect-base';
+import '../ids-tag/ids-tag';
+
+// Import Sass to be encapsulated in the component shadowRoot
+import styles from './ids-multiselect.scss';
+import { stringToBool, stringToNumber } from '../../utils/ids-string-utils/ids-string-utils';
+
+/**
+ * IDS Multiselect Component
+ * @type {IdsMultiselect}
+ * @inherits IdsDropdown
+ * @mixes IdsEventsMixin
+ * @mixes IdsThemeMixin
+ * @part container - the container element
+ */
+@customElement('ids-multiselect')
+@scss(styles)
+class IdsMultiselect extends Base {
+  #selectedList:Array<any>;
+
+  constructor() {
+    super();
+    this.#selectedList = [];
+  }
+
+  /**
+   * Invoked each time the custom element is add into a document-connected element
+   */
+  connectedCallback() {
+    this.#populateSelected();
+    Base.prototype.connectedCallback.apply(this);
```

Comment:
`Base.prototype.connectedCallback.apply(this);` is a more "legacy" version of `super.connectedCallback();`. Thinking this would be better:

```suggestion
    super.connectedCallback();
    this.#populateSelected();
```